content/ja/project-management.md

```diff
@@ -89,7 +89,7 @@ menu:
 
 ・・・等、上記以外にも様々なプロジェクトを手懸けております。
 
-### PMアプローチ
+#### PMアプローチ
```

Comment:
If you make this an h4, the section before it should be an h3. Basically, the title of the page is an h1, and the sections next should be h2's, then h3's. Please don't use these just for sizing. They have meaning, to help the search engine understand the "structure" of the page.